hooks/check

```diff
@@ -11,19 +11,32 @@ fi
 
 ok=yes
 vault="secret/$GENESIS_VAULT_PREFIX"
-domain="$(lookup params.static_ip)"
 for cert in nginx/ssl_certificate; do
   if ! safe exists "$vault/$cert"; then
     describe "  #R{✘}  $vault/$cert [#Y{MISSING}]"
     ok=no
   else
-    if safe --quiet x509 validate "$vault/$cert" --for "$(lookup params.static_ip)" >/dev/null 2>&1; then
-      describe "  #G{✔}  $vault/$cert [#G{OK}]"
+    if [[ $(lookup params.external_domain) == $(lookup params.static_ip) ]]; then
```

Comment:
Technically, if external_domain is not set, it will not be the same as static_ip -- it will be "".  But by using the above recommendation, you don't have to check for it being empty because it will then default to the value in $static_ip.

Reply:
What I'm saying is there is no check if the external_domain is "", which is what would be reported if you use `lookup` without specifying a default.  Basically, you would need to check if it was either set to the same thing as static_ip *or* set to empty string, but by using the value of static_ip in the lookup when setting the variable, it would ensure that it was set to static_ip value if not present.

Reply:
Ahh now I understand what you mean. Will fix with the default to static ip suggestion.